SPECS/kernel-alt.spec

```diff
@@ -55,652 +55,642 @@ Requires(posttrans): xcp-python-libs >= 3.0.2-4.2.xcpng8.3
+#Patch367: 0088-fs-dlm-use-READ_ONCE-for-config-var.patch
+#Patch368: 0089-fs-dlm-fix-typo-in-tlv-prefix.patch
+#Patch369: 0090-fs-dlm-clear-CF_APP_LIMITED-on-close.patch
+#Patch370: 0091-fs-dlm-cleanup-and-remove-_send_rcom.patch
+#Patch371: 0092-fs-dlm-introduce-con_next_wq-helper.patch
+#Patch372: 0093-fs-dlm-move-to-static-proto-ops.patch
+#Patch373: 0094-fs-dlm-introduce-generic-listen.patch
+#Patch374: 0095-fs-dlm-auto-load-sctp-module.patch
+#Patch375: 0096-fs-dlm-generic-connect-func.patch
+#Patch376: 0097-fs-dlm-fix-multiple-empty-writequeue-alloc.patch
+#Patch377: 0098-fs-dlm-move-receive-loop-into-receive-handler.patch
+#Patch378: 0099-fs-dlm-implement-delayed-ack-handling.patch
+#Patch379: 0100-fs-dlm-fix-return-EINTR-on-recovery-stopped.patch
+#Patch380: 0101-fs-dlm-avoid-comms-shutdown-delay-in-release_lockspa.patch
+#Patch381: 0001-bpf-Do-not-use-ax-register-in-interpreter-on-div-mod.patch
+#Patch382: 0002-bpf-Fix-32-bit-src-register-truncation-on-div-mod.patch
+#Patch383: 0003-bpf-Fix-truncation-handling-for-mod32-dst-reg-wrt-ze.patch
+Patch384: 0001-x86-timer-Skip-PIT-initialization-on-modern-chipsets.patch
+Patch385: 0001-x86-timer-Force-PIT-initialization-when-X86_FEATURE_.patch
+Patch386: 0001-x86-timer-Don-t-skip-PIT-setup-when-APIC-is-disabled.patch
+Patch387: 0001-nbd-Fix-use-after-free-in-pid_show.patch
+# GFS2/DLM not supported
+#Patch388: 0001-fs-dlm-remove-check-SCTP-is-loaded-message.patch
+#Patch389: 0001-fs-dlm-let-handle-callback-data-as-void.patch
+#Patch390: 0001-fs-dlm-remove-double-list_first_entry-call.patch
+#Patch391: 0001-fs-dlm-don-t-call-kernel_getpeername-in-error_report.patch
+#Patch392: 0001-fs-dlm-replace-use-of-socket-sk_callback_lock-with-s.patch
+#Patch393: 0001-fs-dlm-fix-build-with-CONFIG_IPV6-disabled.patch
+#Patch394: 0001-fs-dlm-check-for-pending-users-filling-buffers.patch
+#Patch395: 0001-fs-dlm-remove-wq_alloc-mutex.patch
+#Patch396: 0001-fs-dlm-memory-cache-for-writequeue_entry.patch
+#Patch397: 0001-fs-dlm-memory-cache-for-lowcomms-hotpath.patch
+#Patch398: 0001-fs-dlm-print-cluster-addr-if-non-cluster-node-connec.patch
+Patch399: 0001-xen-x86-obtain-upper-32-bits-of-video-frame-buffer-a.patch
+Patch400: 0001-xen-x86-obtain-full-video-frame-buffer-address-for-D.patch
+# GFS2/DLM not supported
+#Patch401: 0001-dlm-uninitialized-variable-on-error-in-dlm_listen_fo.patch
+#Patch402: 0001-dlm-add-__CHECKER__-for-false-positives.patch
+#Patch403: 0001-fs-dlm-fix-grammar-in-lowcomms-output.patch
+#Patch404: 0001-fs-dlm-fix-race-in-lowcomms.patch
+#Patch405: 0001-fs-dlm-relax-sending-to-allow-receiving.patch
+#Patch406: 0001-fs-dlm-fix-sock-release-if-listen-fails.patch
+#Patch407: 0002-fs-dlm-retry-accept-until-EAGAIN-or-error-returns.patch
+#Patch408: 0003-fs-dlm-remove-send-repeat-remove-handling.patch
+#Patch409: 0001-xen-pvh-set-xen_domain_type-to-HVM-in-xen_pvh_init.patch
+#Patch410: 0001-xen-pvh-correctly-setup-the-PV-EFI-interface-for-dom.patch
```

Comment:
Ok, you know what I'm going to ask here.

Reply:
Fixed